api-admin/src/main/java/bio/terra/pearl/api/admin/service/siteContent/SiteContentExtService.java

```diff
@@ -76,7 +76,14 @@ public SiteContent create(
     Portal portal = authUtilService.authUserToPortal(user, portalShortcode);
     siteContent.setPortalId(portal.getId());
     siteContent.setStableId(stableId);
-    return siteContentService.createNewVersion(siteContent);
+    SiteContent newSiteContent = siteContentService.createNewVersion(siteContent);
+
+    return loadSiteContent(
+            newSiteContent.getId(),
+            portalEnvironmentService
+                .findOne(portal.getShortcode(), EnvironmentName.sandbox)
+                .orElseThrow(() -> new IllegalStateException("Portal environment not found")))
+        .orElseThrow(() -> new IllegalStateException("Failed to load created site content"));
```

Comment:
I haven't read through all of the code yet, but to speed up my understanding, I'm curious why this extra load is needed now

Reply:
Ah, I was able to fix it, so I reverted this back to the same as before.

That being said, since on 'create' it goes through a separate code path for attaching all of the children, it was really easy to get out of date and out of sync with the main `loadSiteContent` code path. I'm not sure maintaining both code paths is ideal, so I'm slightly in favor of doing it like I had done - creating the object, then calling "loadSiteContent". But since both  paths are working now I don't feel strongly.